elasticsearch_dsl/utils.py

```diff
@@ -101,6 +101,8 @@ def __getstate__(self):
     def __setstate__(self, state):
         self._l_, self._obj_wrapper = state
 
+    def to_list(self):
+        return [x for x in self.__iter__()]
```

Comment:
I suggest doing it analogically with AttrDict. 

This will get the original value.
```
def to_list(self):
    return self._l_
```